modules/serverless-functions-automatic-build-deploy.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * functions/serverless-functions-on-cluster-builds.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-functions-automatic-build-deploy_{context}"]
+= Configuring automatic building and deployment on GitHub Updates
+
+You can use the Shipwright framework to configure a function to be automatically built and deployed on the cluster whenever the source code in its GitHub repository is updated.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed the Knative (`kn`) CLI.
```

Comment:
Just a nit. Does it make sense to put it like this to have the same format for both OpenShift cli and Knative CLI:
* You have installed the Knative CLI (`kn`).